src/components/CreateFamily/FamilyForm/CreateFamilyModal.cy.tsx

```diff
@@ -0,0 +1,63 @@
+import { noop } from '@fullstacksjs/toolbox';
+
+import { CreateFamilyModal } from './CreateFamilyModal';
+
+const requiredFieldMessage = 'این فیلد ضروری است';
+const minLengthMessage = 'نام خانواده باید حداقل ۵ حرف باشد';
+const buttonSelector = '[data-test="submit-button"]';
+const inputSelector = '[data-test="family-name"]';
+
+describe('CreateFamilyModal', () => {
+  beforeEach(() => {
+    cy.mount(<CreateFamilyModal opened onClose={noop} />);
+  });
+  it('should contains text input to enter family name', () => {
+    cy.get('form').get(inputSelector).should('match', 'input');
+  });
+  it('should contains submit button to create family', () => {
+    cy.get('form')
+      .get(buttonSelector)
+      .should('have.attr', 'type')
+      .and('equal', 'submit');
+  });
+  it('should not show a required error message when family name is not empty', () => {
+    cy.get('form').within(() => {
+      cy.get(inputSelector);
```

Comment:
according to your title, you have to type something in your input

Reply:
That's true.